utils/check-missing-logos.js

```diff
@@ -0,0 +1,46 @@
+const {
+  readQuickstartFile,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+/**
+ * Validate that a quickstart has a logo image
+ * @param {String[]} quickstartDirs - The directories of the quickstarts
+ * @returns {String[]} - The directories of the quickstarts that do not have a logo image
+ */
+const validateLogoExists = (quickstartDirs) => {
+  const noLogos = quickstartDirs
+    .map((quickstart) => {
+      const quickstartConfig = readQuickstartFile(quickstart).contents[0];
+      
+      if (!quickstartConfig.logo) {
+        return quickstart.match(/newrelic-quickstarts(.*)/)[1];
+      }
+      
+      
+    }).filter(Boolean);
+  return noLogos;
```

Comment:
Ye that makes a lot more sense, not sure why i went with the `map` here. We will need to use `config.logo` because some of the configs have `logo: ''` so we need to account for those